decidim-comments/lib/decidim/comments/comment_serializer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+module Decidim
+  module Comments
+    class CommentSerializer < Decidim::Exporters::Serializer
+      # Serializes a comment
+      def serialize
+        {
+          id: resource.id,
+          created_at: resource.created_at,
+          body: resource.body,
+          author: {
+            id: resource.author.id,
+            name: resource.author.name
+          },
+          alignment: resource.alignment,
+          depth: resource.depth,
+          user_group: {
+            id: resource.user_group.try(:id),
+            name: resource.user_group.try(:name)
+          },
+          commentable_id: resource.decidim_commentable_id,
+          commentable_type: resource.decidim_commentable_type
```

Comment:
Not sure that's feasible, but will try. The difficult thing about it is we don't necessarily know how to construct urls for a commentable resource.